frontend/src/templates/vrImageTemplate.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { graphql } from 'gatsby';
+import { Typography } from '@material-ui/core';
+import loadable from '@loadable/component';
+
+const VRImage = loadable(() => import('./AFrameWrapper'));
```

Comment:
what does loadable do vs a regular import?

Reply:
Resolved, loadable is not needed